### ISSUE TITLE
subresource status is Rejected, but cli for vs still shows Accepted

### ISSUE THREAD

**Issue by ilackarms** [solo-io/gloo#669]
we need to make sure that table printing virtual service propagates the subresource errors for the user
currently this yaml:

```
displayName: coalmine
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"gateway.solo.io/v1","kind":"VirtualService","metadata":{"annotations":{},"name":"coalmine","namespace":"gloo-system"},"spec":{"displayName":"coalmine","virtualHost":{"domains":["*"],"name":"gloo-system.coalmine","routes":[{"matcher":{"prefix":"/findPets"},"routeAction":{"single":{"destinationSpec":{"rest":{"functionName":"findPets","parameters":{}}},"upstream":{"name":"default-petstore-v2-8080","namespace":"gloo-system"}}}},{"matcher":{"prefix":"/findPetWithId"},"routeAction":{"multi":{"destinations":[{"destination":{"destinationSpec":{"rest":{"functionName":"findPetById","parameters":{"headers":{":path":"/findPetWithId/{id}"}}}},"upstream":{"name":"default-petstore-v2-8080","namespace":"gloo-system"}},"weight":50},{"destination":{"destinationSpec":{"rest":{"functionName":"findPetById","parameters":{"headers":{":path":"/findPetWithId/{id}"}}}},"upstream":{"name":"default-petstore-v1-8080","namespace":"gloo-system"}},"weight":50}]}}},{"matcher":{"prefix":"/petstore"},"routeAction":{"single":{"upstream":{"name":"default-petstore-v1-8080","namespace":"gloo-system"}}},"routePlugins":{"prefixRewrite":{"prefixRewrite":"/api/pets"}}}]}}}
  labels: {}
  name: coalmine
  namespace: gloo-system
  resourceVersion: "235177"
status:
  reportedBy: gateway
  state: Accepted
  subresourceStatuses:
    '*v1.Proxy gloo-system gateway-proxy':
      reason: "2 errors occurred:\n\t* route_config.invalid route: list did not find
        upstream gloo-system.my-aws\n\t* route_config.plugin error on route: {my-aws
        gloo-system} is not an AWS upstream\n\n"
      reportedBy: gloo
      state: Rejected
```

will print as this table 

```
+-----------------+--------------+---------+------+----------+---------+--------------------------------+
| VIRTUAL SERVICE | DISPLAY NAME | DOMAINS | SSL  |  STATUS  | PLUGINS |             ROUTES             |
+-----------------+--------------+---------+------+----------+---------+--------------------------------+
| coalmine        | coalmine     | *       | none | Accepted |         | /helloworld -> my-aws          |
```

**Comment by rickducott** [comment]
This was just addressed in #651

**Comment by ilackarms** [comment]
whoops! this was based on a user report and not on latest `master`. closing
